source/administration/config-settings.rst

```diff
@@ -2345,6 +2359,18 @@ Id Attribute
 | This feature's ``config.json`` setting is ``"IdAttribute": ""`` with string input.                                                                                   |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Guest Attribute
+^^^^^^^^^^^^^^^^^
+(Optional) The attribute in the SAML Assertion used to apply a guest role to users in Mattermost.
```

Comment:
Wondering if we should say "The attribute and value in the SAML...". And potentially provide a sample. 
ie. "IsGuest=true" or "UserType=Guest".

Reply:
That works for me!